lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -301,6 +307,31 @@ func getHEConfig(gwConfig *lte_models.GatewayHeConfig) *lte_mconfig.PipelineD_HE
 		EncryptionKey:          gwConfig.EncryptionKey,
 	}
 }
+
+func getMMEPoolConfigs(networkID string, poolingConfig lte_models.CellularGatewayPoolRecords, cellGateway configurator.NetworkEntity, graph configurator.EntityGraph) (*lte_models.CellularGatewayPoolRecord, uint32, error){
```

Comment:
putting a comment before the function to specify the default return values would be nice.

Reply:
+1. Alternative is named returns. I doc comment except when named returns also serve alternative purpose